### ISSUE TITLE
Speaker Identification

### ISSUE THREAD

**Issue by linksyncjameshwartlopez** [tyiannak/pyAudioAnalysis#119]
Should i be able to do speaker identification using this library?

**Comment by eduamf** [comment]
I'm working with this same goal!

**Comment by StanSilas** [comment]
Have @linksyncjameshwartlopez  @eduamf  Please share code , in case you achieve the object. Thanks in advance

**Comment by channae** [comment]
Can we do speaker identification using this library? I'm also looking for a library which can do speaker identification. Appreciate your inputs.

**Comment by piyust54** [comment]
yes speaker recognition, speaker identification..everything can be done..

**Comment by linksyncjameshwartlopez** [comment]
https://pypi.org/project/bob.bio.spear/

I think that's worth sharing here. :)

**Comment by channae** [comment]
@piyust54 Thanks for the reply. Are there any documentation or sample code available about implementation or usage of text independent speaker identification? I could only find speaker diarization related usage in the readme.

**Comment by piyust54** [comment]
https://github.com/subedisuraj/speaker-recognition
